### ISSUE TITLE
Declaration

### ISSUE THREAD

**Issue by ebeshero** [markupdeclaration/scratchpad#1]
@sydb suggests this Declaration be one of independence for our data. I wonder if it's also to be a declaration about declarative programming.

**Comment by ndw** [comment]
That wasn't first among my thoughts about declarative markup. Do you have some specific examples in mind?

**Comment by sydb** [comment]
To be clear, I was more rejecting the notion that our use of “declaration” was akin to a declaration of war than asserting it should be a declaration of independence.

**Comment by liamquin** [comment]
For me it's always been a strength of XML that the culture (and the difficulty of data binding to traditional languages) has kept central the idea that document authors/editors/transcribers/creators own their data and the format, not application developers. The information is independent of any one program, any one vendor. Is that what you meant by independence, @sydb ?

**Comment by sydb** [comment]
Absolutely, @liamquin; although not really what I meant for “declaration”, (since I use “descriptive” to describe markup of that sort).

But I don’t think it’s useful at this point to hash out what I (or someone else) meant at the time. As long as we’re not thinking “declaration of war”, we should just move forward. :-)
